src/main/java/com/microsoft/jenkins/azuread/AzureSecurityRealm.java

```diff
@@ -188,7 +189,7 @@ private ClientSecretCredential getClientSecretCredential() {
         if (JenkinsJVM.isJenkinsJVM()) {
             ProxyConfiguration proxyConfiguration = Jenkins.get().getProxy();
             if (proxyConfiguration != null && StringUtils.isNotBlank(proxyConfiguration.getName())) {
-                Proxy proxy = proxyConfiguration.createProxy("graph.microsoft.com");
+                Proxy proxy = proxyConfiguration.createProxy("https://graph.microsoft.com");
```

Comment:
this is intentionally without a domain, it's used for creating a proxy

Reply:
Sorry, my rebase was not clean. Not part of the fix, let me update this PR.